docs/_tutorials/phpqa-tutorial.md

```diff
@@ -30,6 +30,17 @@ Start PHPTesting!!!
 
 ## How to use it?
 
+### Create a git repository
+
+The `phpqa` tool expects you to be operating in the context of a `git`
+repository. So before continuing, you'll want to decide where you'll be storing
+your test files, and create a `git` repo in that directory.
+
+It is also important that you have a `phpt/` directory within that repo.
+By default, `phpqa` places tests that it generates in the `phpt/` directory and
+will throw an error if the directory does not exist. You can override this
+behavior as described below.
```

Comment:
Early versions of `phpqa` were indeed with `phpt` as default directory, but this is not required since a couple of weeks. Could you please make sure that you have the latest version of the tool? Also, it would be really great if you take time to open an issue at the [docker-phpqa](https://github.com/herdphp/docker-phpqa/tree/master) repository describe your problem for us.